web/documentserver-example/python/.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-complexity = 15
+max-line-length = 120
+per-file-ignores = __init__.py:F4
```

Comment:
Could you please explain to me what `per-file-ignores` means?

Reply:
> Could you please explain to me what `per-file-ignores` means?

`per-file-ignores = A:B` will ignore `B` rules in `A` file(s). F4 codes is about importing modules, but flake8 is not friendly to init files. For example it will always trow F401(unused import).